docker/Dockerfile

```diff
@@ -124,13 +124,17 @@ USER opal
 # CEDAR CLIENT IMAGE --------------------------------
 # Using standalone image as base --------------------
 # ---------------------------------------------------
+# Extract the Cedar Agent binary from the existing image
+FROM permitio/opal-client-cedar:latest AS cedar-extractor
```

Comment:
We can't extract Cedar Agent for OPAL from OPAL, we need to get it from Cedar Agent.
Either download the binary from the Github release https://github.com/permitio/cedar-agent/releases
or extract the binary from the Docker hub version https://hub.docker.com/r/permitio/cedar-agent/tags
We need to specify explicitly which Cedar Agent version to use. For now, set to the latest (0.2.0)